src/packages/client/package.json

```diff
@@ -113,6 +113,7 @@
     "strip-indent": "3.0.0",
     "ts-jest": "26.4.4",
     "ts-node": "9.1.1",
+    "ts-toolbelt": "^8.0.7",
```

Comment:
It seems like you just extracted the utilities you wanted from ts-toolbelt. Do we actually use this anywhere?